### PR TITLE
TDD project

### PR DESCRIPTION
**Pull request by akos29**
### TDD project

In this request majorly :

- [ ] add `Solver` class
- [ ] add test examples for the `factorial` method
- [ ] add test examples for `reverse` word method 
- [ ] add test examples for `fizzbuzz`
- [ ] add code for the `factorial` 
- [ ] add code for the `reverse`
- [ ] add code for the `fizzbuzz`

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -119,71 +113,50 @@ After you're finished please remove all the comments and instructions!
...
-Install this project with:
-
-<!--
-Example command:
+- Install this project with:
 
 ```sh
-  cd my-project
-  gem install
+  cd shcool-library-app
```
Comment (DeliceLydia) [inline README.md]: - [ ] Please add the name of the project you are working on `TDD project`

File: README.md [hunk 2]
```diff
@@ -119,71 +113,50 @@ After you're finished please remove all the comments and instructions!
...
 
 To run the project, execute the following command:
 
-<!--
-Example command:
-
 ```sh
-  rails server
+  ruby main.rb
+
```
Comment (DeliceLydia) [inline README.md]: - [ ] I think we are not using this anywhere right? you can remove it

**Review by DeliceLydia — CHANGES_REQUESTED** [review]
## Hi Team, 

Good job so far! There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- [ ] Awesome PR description
- [ ] Tested all methods

## Required Changes ♻️
_Check the comments under the review._



### Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by rustam-tolipov — APPROVED** [review]
Hi Team, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Developer-512px-10

## Highlights
1. No Linters errors 💯
2. All requirements are complete 💯
3. Great readme template 💯 

## Optional suggestions
No optional suggestions :)

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**